src/App.css

```diff
@@ -2,4 +2,34 @@
 
 * {
     font-family: 'Montserrat', sans-serif;
+}
+
+/* header */
+
+header {
```

Comment:
Using global styles is not recommended for real-world applications. You will ultimately end up with naming conflicts (think about logo class inside header and footer) or you'll need something old-school like BEM.

Have a look at css-modules or (even better IMO) CSS-in-JS like styled-components. Then you can move the styles next to the Header component. Makes it much cleaner and easier to maintain when the app grows